src/docker-compose/docker-compose-render.ts

```diff
@@ -20,12 +20,13 @@ interface DockerComposeFileServiceSchema {
   readonly dependsOn?: string[];
   readonly build?: DockerComposeBuild;
   readonly image?: string;
-  readonly command?: string[];
+  readonly command?: string | string[];
```

Comment:
```suggestion
  readonly command?: string[];
```

We can't have union types due, they don't work in other languages.